django_project/realtime/tasks/ash.py

```diff
@@ -32,6 +57,40 @@ def check_processing_task():
         result = AsyncResult(id=task_id, app=realtime_app)
         ash.task_status = result.state
         ash.save()
+    # Checking analysis task
+    for ash in Ash.objects.exclude(
```

Comment:
It is better to separate all these. It is confusing to see it now.

Reply:
Yes, agree... we need to consider also this periodic checking's duration, since it will affect analysis duration. Is it possible to put signal for celery task? For example, if task_a.success.connect(task_b) *or may be this is chain task?